templates/eks/components/eks-data-load-function-stack-06.yml

```diff
@@ -1,55 +1,55 @@
----
-AWSTemplateFormatVersion: '2010-09-09'
-Parameters:
-  RoleARN:
-    Type: String
-  MyVPC:
-    Type: AWS::EC2::VPC::Id
-  # DBSubnet1:
-  #   Type: AWS::EC2::Subnet::Id
-  # DBSubnet2:
-  #   Type: AWS::EC2::Subnet::Id
-  # DBSubnet3:
-  #   Type: AWS::EC2::Subnet::Id
-  PrivateSubnets:
-    Type: String
-Resources:
-  DataLoadFunction:
-    Type: AWS::Lambda::Function
-    Properties:
-      Handler: lambda_function.lambda_handler
-      MemorySize: 3008
-      Code:
-        S3Bucket: studio-eks
-        S3Key: lambda/DataLoadFunction.zip
-      Role:
-        Ref: RoleARN
-      Runtime: python3.7
-      Timeout: 900
-      VpcConfig:
-        SecurityGroupIds:
-        - Ref: LambdaSecurityGroup
-        SubnetIds:
-          Fn::Split:
-          - ","
-          - !Ref PrivateSubnets
-  LambdaSecurityGroup:
-    Type: AWS::EC2::SecurityGroup
-    Properties:
-      GroupDescription: Group for Lambda Instance
-      VpcId:
-        Ref: MyVPC
-      SecurityGroupIngress:
-      - IpProtocol: tcp
-        FromPort: 5432
-        ToPort: 5432
-        CidrIp: 10.0.0.0/16
-Outputs:
-  FunctionARN:
-    Description: FunctionARN
-    Value:
-      Fn::GetAtt:
-      - DataLoadFunction
-      - Arn
-    Export:
-      Name: FunctionARN
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Parameters:
+  RoleARN:
+    Type: String
+  MyVPC:
+    Type: AWS::EC2::VPC::Id
+  # DBSubnet1:
+  #   Type: AWS::EC2::Subnet::Id
+  # DBSubnet2:
+  #   Type: AWS::EC2::Subnet::Id
+  # DBSubnet3:
+  #   Type: AWS::EC2::Subnet::Id
+  PrivateSubnets:
+    Type: String
+Resources:
+  DataLoadFunction:
+    Type: AWS::Lambda::Function
+    Properties:
+      Handler: lambda_function.lambda_handler
+      MemorySize: 3008
+      Code:
+        S3Bucket: curriki-nested-cft

```

Comment:
Why do we need to do that it was due to permission issue?
Can we take this input from parameters in order to fix permission issue?